floodwatch/src/js/components/Compare.js

```diff
@@ -0,0 +1,161 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import $ from 'jquery';
+import {FilterParent} from './FilterParent';
+import AdBreakdown from '../../stubbed_data/adbreakdown_response.json';
+import Filters from '../../stubbed_data/filter_response.json';
+
+// import '../../Compare.css';
+
+export class Compare extends Component {
+  render() {
+    return (
+      <Row>
+        <Col xs={12}>
+        <CompareContainer/>
+        </Col>
+      </Row>
+    )
+  }
+}
+
+type StateType = {
+  leftOptions: Object,
```

Comment:
These need types, but I think you can use the stuff that's now on develop - we can do it post-merge, which it looks like we need to do by hand now that I made some breaking changes.